### PR TITLE
CAMEL-18909: DTO generator option in camel-jbang generate command

### PR DESCRIPTION
**Pull request by mgubaidullin**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]
I think this should go also on camel-3.x and main

**Comment by oscerd** [comment]
We can do that later once this one has been merged. The comment was just to not forget about it.

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
Yeah a new PR against camel-3.x branch as that should be not be jakarta ee based so we cannot cherry pick.

**Comment by oscerd** [comment]
Ah yeah, i have to start to remind about Jakarta compliancy :-)

File: dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/CodeRestGenerator.java [hunk 1]
```diff
@@ -80,4 +102,34 @@ private JsonNode readNodeFromYaml() throws FileNotFoundException {
...
+
+        final CodegenConfigurator configurator = new CodegenConfigurator()
+                .setGeneratorName(GENERATOR_NAME)
+                .setLibrary(LIBRARY)
+                .setInputSpec(input)
+                .setModelPackage(packageName)
+                .setAdditionalProperties(
+                        Map.of(
+                                SERIALIZABLE_MODEL, "false",
+                                "useJakartaEe", "true",
```
Comment (davsclaus) [inline dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/CodeRestGenerator.java:120]: this should be false as camel 3.x is not jakarta

**Review by davsclaus — CHANGES_REQUESTED** [review]

**Comment by davsclaus** [comment]
okay we can cherry pick this to camel-3.x

**Comment by davsclaus** [comment]
I can do the change as Marat is on PTO

**Pull request merged**